src/main/java/Duke.java

```diff
@@ -1,10 +1,80 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
 public class Duke {
+    public static ArrayList<Task> tasks = new ArrayList<Task>();
```

Comment:
Maybe you can try using camelCase when naming array lists as they are variables.